src/main/java/org/example/Wrapper.java

```diff
@@ -0,0 +1,111 @@
+package org.example;
+
+import org.example.Operations.*;
+import org.example.Util.IrodsException;
+import org.example.Util.Response;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.util.Base64;
+
+public class Wrapper {
```

Comment:
Open an issue to rename this class.

We'll handle that in a separate PR.

Reply:
Got it, issue created.

Reply:
Issue created.